Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM python:3.9.7-buster
+FROM python:3.10.8-buster
```

Comment:
Can this be changed to 3.10.4 that matches setup.py?

Reply:
done